### PR TITLE
✨ Add fela-plugin-autoprefixer

### PR DESCRIPTION
**Pull request by filip-dot-cube**
This MR add vendor prefixes

### CODE REVIEW

File: gatsby-theme-fela/package.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
     "fela": "^11.0.2",
     "fela-monolithic": "^11.0.2",
     "fela-plugin-expand-shorthand": "^11.0.2",
+    "fela-plugin-fallback-value": "^11.3.0",
```
Comment (Tomas2D) [inline gatsby-theme-fela/package.json:14]: I think that plugins should match the version which fela base package uses. What other thinks?

Comment (horaklukas) [reply gatsby-theme-fela/package.json:14]: Agree, we should keep consistent version for all fela ecosystem, it's a common practice

Comment (filip-dot-cube) [reply gatsby-theme-fela/package.json:14]: Ill downgrade this, but isnt it just the plugin version, that does not necessarily reffer to fela version?

Comment (Tomas2D) [reply gatsby-theme-fela/package.json:14]: I do not think so. I remember that I had problem with that.

Comment (horaklukas) [reply gatsby-theme-fela/package.json:14]: The versioning policy of Fela is that bump of a version of any package cause automatically bump of all other packages (see https://github.com/lerna/lerna#fixedlocked-mode-default). There is no problem to have all fela packages at the same version actually it's a only way to ensure they'll work together. So either downgrade plugin or upgrade other fela packages.

Comment (filip-dot-cube) [reply gatsby-theme-fela/package.json]: ✅

**Pull request merged**